docs/guides/first-app.mdx

```diff
@@ -124,173 +120,182 @@ With a schema designed, we can now move on using our client to to apply that sch
 ]}>
   <TabItem value="shell">
 
-```sh
-zed schema write <(cat << EOF
-definition blog/user {}
-
-definition blog/post {
-	relation reader: blog/user
-	relation writer: blog/user
-
+<SampleCodeBlock lang="shell">
+{`zed schema write <(cat << EOF
+definition {{ tenant }}/user {}
+·
+definition {{ tenant }}/post {
+	relation reader: {{ tenant }}/user
+	relation writer: {{ tenant }}/user\n
 	permission read = reader + writer
 	permission write = writer
 }
 EOF
-)
-```
+)`}
+</SampleCodeBlock>
 
   </TabItem>
   <TabItem value="go">
 
-```go
-package main
-
+<SampleCodeBlock lang="go">
+{`package main
+·
 import (
 	"context"
 	"log"
 	"os"
-
+·
 	pb "github.com/authzed/authzed-go/proto/authzed/api/v1"
 	"github.com/authzed/authzed-go/v1"
 	"github.com/authzed/grpcutil"
 )
-
-const schema = `definition blog/user {}
-
-definition blog/post {
-	relation reader: blog/user
-	relation writer: blog/user
-
+·
+const schema = \`definition {{ tenant }}/user {}
+·
+definition {{ tenant }}/post {
+	relation reader: {{ tenant }}/user
+	relation writer: {{ tenant }}/user\n
 	permission read = reader + writer
 	permission write = writer
-}`
-
+}\`
+·
 func main() {
+  {% if not authzed %}
+      // NOTE: to connect to a SpiceDB behind TLS, use grpcutil.WithBearerToken
+        // add grpcutil.WithSystemCerts(grpcutil.VerifyCA)
+  {% endif %}
 	client, err := authzed.NewClient(
-		"grpc.authzed.com:443",
-		grpcutil.WithBearerToken("t_your_token_here_1234567deadbeef"),
-		grpcutil.WithSystemCerts(grpcutil.VerifyCA),
+		"{{ endpoint }}",
+    {% if authzed %}
+            grpcutil.WithBearerToken("{{ token }}"),
+	        grpcutil.WithSystemCerts(grpcutil.VerifyCA),
+    {% else %}
+    	grpcutil.WithInsecureBearerToken("{{ token }}"),
```

Comment:
Hmm? We do so for all the languages, and add a comment for each

Reply:
I meant so that the rendered form for SpiceDB looks something like this:

```go
client, err := authzed.NewClient(
  "localhost:50051",
  // Swap with commented options below to enable/disable TLS
  grpc.WithTransportCredentials(insecure.NewCredentials()),
  grpcutil.WithInsecureBearerToken("t_your_token_here_1234567deadbeef"),
  // grpcutil.WithSystemCerts(grpcutil.VerifyCA),
  // grpcutil.WithBearerToken(t_your_token_here_1234567deadbeef),
)
```

I think you also need to add the missing insecure line

Reply:
Its a bit long though; that's why I just put the comment in. @samkim, thoughts?

Reply:
I prefer inline over the note in a comment. For these examples, we've been trying to keep them as close to copy/paste-able as possible and the inline commented out params are closer to that aim

Reply:
Done